### DISCUSSION TITLE
Support for Camera Raw Files

### DISCUSSION THREAD

**Discussion by crittercam** [coderaiser/cloudcmd#353] [Ideas]
I'm hoping it might be possible to add support for Camera Raw files, so files from a camera could be opened similarly to a jpg or mp4 files.  There is a raw library available, but I'm not sure how it would be implemented.  The library is hosted at https://www.libraw.org/.

**Comment by coderaiser** [comment]
Looks like this library written on `c++` how it should help :)?

**Reply by crittercam** [reply]
I should say I was hoping it will help.  What I'm trying to do is treat the native files from my camera like a standrad file, such as jpg or mp4.
